pkgs/by-name/ti/tiddit/package.nix

```
@@ -0,0 +1,75 @@
+{
+  lib,
+  stdenv,
+  bwa,
+  zlib,
+  python3,
+  fetchFromGitHub,
+  python3Packages,
+}:
+let
+  fermi2 = stdenv.mkDerivation {
+    pname = "fermi2";
+    version = "unstable-20210521";
+
+    src = fetchFromGitHub {
+      owner = "lh3";
+      repo = "fermi2";
+      rev = "cb1410972b2bd330883823116931ae67ead8b30f";
+      hash = "sha256-jDn1OBuGWDMEHI5A3R9meOykEGM6yjItSnUpx36DxgA=";
+    };
+
+    buildInputs = [ zlib ];
+    installPhase = ''
+      mkdir -p $out/bin
+      cp fermi2 $out/bin
+    '';
+  };
+  ropebwt2 = stdenv.mkDerivation {
+    name = "ropebwt2";
+    version = "unstable-20210201";
+
+    src = fetchFromGitHub {
+      owner = "lh3";
+      repo = "ropebwt2";
+      rev = "bd8dbd3db2e9e3cff74acc2907c0742c9ebbf033";
+      hash = "sha256-R/VvbprwcfXF2TBZOYmc1MU3AzCcXFfWCHoYYumXtI8=";
+    };
+
+    buildInputs = [ zlib ];
+    installPhase = ''
+      mkdir -p $out/bin
+      cp ropebwt2 $out/bin
+    '';
+  };
+in
+python3Packages.buildPythonApplication {
+  name = "tiddit";
+  version = "3.6.1";
+  src = fetchFromGitHub {
```

```suggestion
  version = "3.6.1";
  pyproject = true;

  src = fetchFromGitHub {
```

and add later `build-system = ...`

---

What is the advantage of pyproject here, where there is only a setup.py ? The manual says it default to setuptools. Thanks,